discovery/gossiper.go

```diff
@@ -1096,23 +1103,28 @@ func (d *AuthenticatedGossiper) networkHandler() {
 					return
 				}
 
-				// Process the network announcement to determine if
-				// this is either a new announcement from our PoV
-				// or an edges to a prior vertex/edge we previously
-				// proceeded.
-				emittedAnnouncements := d.processNetworkAnnouncement(
-					announcement,
-				)
+				// Process the network announcement to
+				// determine if this is either a new
+				// announcement from our PoV or an edges to a
+				// prior vertex/edge we previously proceeded.
+				emittedAnnouncements :=
```

Comment:
This isn't documented anywhere, but style wise, I think we should prefer just leaving it on the same line. Otherwise, by adding a new line here, it just looks awkward and is IMO, harder to parse visually.